ui/npm-content/build.sh

```diff
@@ -65,7 +65,7 @@ echo "Building Monaco editor..."
 yarn build:monaco
 echo "Monaco editor build complete"
 
-# jquery
+# jquery - custom build for guest
```

Comment:
Since we now have another purpose for this and we have it in node modules, does the npm package have everything needed to execute the build? i.e. do we still need to to git pull it separately?

Reply:
No it doesn't have everything, `Gruntfile.js` is not available, and when I tried to copy it and build, it started failing due to some other missing files.